.docker/docker-compose.base.yml

```diff
@@ -16,6 +16,7 @@ services:
       - TYPO3_CONTEXT=Development/BootstrapGrids/Docker
     volumes:
       - '../:/var/www/html'
+      - './php.ini:/usr/local/etc/php/conf.d/custom.ini'
```

Comment:
not sure if this really works like this. i think you should copy this in Dockerfile at build-time. I can fix that. Volumes are only mounted on run-time, which might be to late. At least for apache config it would be to late.